sound/hybrid_font.h

```diff
@@ -548,6 +548,28 @@ class HybridFont : public SaberBase {
     return current_effect_length_;
   }
 
+	void SB_LowBatt() override
+	{
+		STDOUT.println(" ");
+		STDOUT.println("|| SB_LowBatt function start");
+		STDOUT.println(" ");
+
+		//play the fonts low battery sound if it exists.  Otherwise try to play a wave file on the root of the SD Card
+		if (SFX_lowbatt.files_found()) {
+			PlayCommon(&SFX_lowbatt);
+		} else {
+			MountSDCard();
```

Comment:
Let's remove all of this.
If we want to support a global location for files, we should just add it to the effect class so that it can be used for any and all files.